pkg/runtime/runtime.go

```diff
@@ -111,6 +116,12 @@ type TopicRoute struct {
 	routes map[string]Route
 }
 
+// Wrapper to PubSub interface to accommodate retry logic settings
+type pubSub struct {
```

Comment:
Please, name it something else, just relying on the case sensitivity hurts the readability.

Reply:
Would the below be ok?

type pubSubComponent struct {
	retrySettings retry.Settings
	pubSubClient       pubsub.PubSub
}

Reply:
pubSubWrapper - `Component` is an overloaded term in Dapr already.

Reply:
Done.